src/main/apisix/Chart.yaml

```diff
@@ -0,0 +1,49 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+apiVersion: v2
+description: Apache APISIX is a dynamic, real-time, high-performance API gateway, based on the Nginx library and etcd.
+icon: https://apisix.apache.org/img/logo.png
+maintainers:
+- email: liling@apache.org
+  name: liling
+name: apisix
+home: https://apisix.apache.org/
+dependencies:
+- name: etcd
+  version: 0.1.0
```

Comment:
please upgrade to `0.1.1` to avoid failed test jobs on KubeSphere UI.

Reply:
done